styles.css

```diff
@@ -8,6 +8,111 @@
   scroll-behavior: smooth;
 }
 
+/* popup window start */
+.popup {
+  position: absolute;
```

Comment:
- [ ]  The rule `position: absolute;`  appears 8 times in your code, and` display: none;` appears 15 times. This is a lot of repetition. Please write DRY CSS code. If you want to create a CSS rule for multiple elements, use a class attribute instead. For example, instead of using color: #3498db; 3 times 😄 
```
    #intro1 {color: #3498db; font-size: 10px; font-weight: bold;}
    header {color: #3498db; font-size: 20px; background-color: green;}
    #banner {color: #3498db; font-size: 30px; background-image: url(images/static.jpg);}

you can create a class and add it to all elements that need to be blue.
```
   .blue {color: #3498db;}

```